api/src/main/java/jakarta/servlet/http/LocalStrings_fr.properties

```diff
@@ -19,12 +19,12 @@
 # Default localized string information
 # Localized for Locale fr_FR
 
-err.cookie_name_is_token=Le nom de cookie \"{0}\" est un \"token\" r�serv�
+err.cookie_name_invalid=Le nom de cookie \"{0}\" est un \"token\" r�serv�
 err.io.short_read=Lecture partielle
 
-http.method_not_implemented=Le m�thode {0} n''est pas d�finie dans la RFC 2068 et n''est pas support�e par l''API Servlet
+http.method_not_implemented=Le m�thode {0} n''est pas d�finie dans la RFC 2068 et n''est pas support�e par l''API Servlet
 
-http.method_get_not_supported=La m�thode HTTP GET n''est pas support�e par cette URL
-http.method_post_not_supported=La m�thode HTTP POST n''est pas support�e par cette URL
-http.method_put_not_supported=La m�thode HTTP PUT n''est pas support�e par cette URL
-http.method_delete_not_supported=La m�thode HTTP DELETE n''est pas support�e par cette URL 
+http.method_get_not_supported=La m�thode HTTP GET n''est pas support�e par cette URL
+http.method_post_not_supported=La m�thode HTTP POST n''est pas support�e par cette URL
+http.method_put_not_supported=La m�thode HTTP PUT n''est pas support�e par cette URL
+http.method_delete_not_supported=La m�thode HTTP DELETE n''est pas support�e par cette URL 
```

Comment:
Please correct this and fix your editor to not anymore cause this in the future.

Reply:
I've fixed the file.